eng/testing/AppleRunnerTemplate.sh

```diff
@@ -39,14 +36,22 @@ while true; do
     fi
 done
 
-XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
-
-dotnet xharness ios test --app="$APP_BUNDLE" \
-    --targets=$TARGET \
+# Restart the simulator to make sure it is tied to the right user session
+XCODE_VERSION=11.4
+XCODE_PATH="/Applications/Xcode${XCODE_VERSION/./}.app"
```

Comment:
hardcoding the Xcode version will make it difficult to keep this updated when new Xcode versions are rolled out, or locally where I just have Xcode.app.

why do we not use `/Applications/Xcode.app`?

Reply:
We have several versions installed side-by-side on our on-prem machines. It is a practice adopted from Xamarin as it is then super clear on which version is used.

You could change this script to get the version from `xcode-select -p`

Reply:
Yes using `xcode-select` would be much better.

Reply:
I will give it a go, and also will stop specifying the iOS version since if we install a new Xcode, it might not have 13.5 available

Reply:
We basically won't be able to target one version of iOS with confidence.. We can also not specify the version and XHarness will grab the latest